css/styles.css

```diff
@@ -44,3 +44,13 @@
 #autoupgradePhpWarningMainIcon {font-size: 3em;}
 #autoupgradePhpWarn .icon-stack-text {color: white;}
 .panel-heading {text-transform: uppercase;}
+.support-yes, .example-yes {text-align: center;}
+td {padding: 0.5rem 0.8rem;font-size: 1.5em;}
```

Comment:
@matthieu-rolland it's not that important, but for the future, remember that it is better to be more precise in the modules and use a selector with a better specificity

Reply:
I don't think I wrote this, but yes good advice indeed 👍